src/components/savings/SavingsListRowEmptyState.js

```diff
@@ -42,7 +43,7 @@ const SavingsListRowEmptyState = ({ onPress }) => {
         size="lmedium"
         weight="bold"
       >
-        $0.00
+        {lang.toCurrency(0)}
```

Comment:
leaving a note for myself here to double check on other currency symbols being hooked up for this functionality